src/app/exercise/exercises/IntervalExercise/intervalExercise.ts

```diff
@@ -133,11 +135,17 @@ export const intervalExercise = () => {
       const randomStartingNoteNumber: NoteNumber = _.random(range.lowestNoteNumber, range.highestNoteNumber - randomIntervalDescriptor.semitones);
       let lowNoteName = toNoteName(randomStartingNoteNumber);
       let highNoteName = toNoteName(randomStartingNoteNumber + randomIntervalDescriptor.semitones);
-      let [startNoteName, endNoteName] = _.shuffle([lowNoteName, highNoteName])
+      let [startNoteName, endNoteName] = _.shuffle([lowNoteName, highNoteName]);
+      let toPlay;
+      if (settings.intervalType === 'melodic')
+        toPlay = [{ notes: startNoteName }, { notes: endNoteName } ]
+      else
+        toPlay = [{ notes: [startNoteName, endNoteName] }]
```

Comment:
```suggestion
      const partToPlay = settings.intervalType === 'melodic' ? 
        [{ notes: startNoteName }, { notes: endNoteName } ] :
        [{ notes: [startNoteName, endNoteName] }];
```

A little bit more elegant.